playpen/ansible/dev-playbook.yml

```diff
@@ -6,4 +6,5 @@
     - core
     - db
     - qpidd
-    - dev
\ No newline at end of file
+    - dev
+    - lazy
```

Comment:
I didn't realize we had a separate dev and vagrant playbook these days. I had actually been planning to rename vagrant-playbook to dev-playbook so now that's confusing.